tidb-server/main.go

```diff
@@ -345,6 +345,12 @@ func loadConfig() string {
 			return err.Error()
 		}
 		terror.MustNil(err)
+	} else {
+		// configCheck should have the config file specified
+		if *configCheck {
+			fmt.Fprintln(os.Stderr, "config check failed", errors.New("no config file specified for config-check"))
+			os.Exit(1)
```

Comment:
Could we use `MustNil`?

Reply:
Em... it's not in test file .
This is same as previous config valid code when tidb start, `Fprintln` to Stderr with err. 
Cause user specified no config file, we new an error here.

Reply:
I know it isn't the test file, you can see `MustNil` is called on line 347.

Reply:
Enen, got it. I've update my comment